packages/react-icon-provider/README.md

```diff
@@ -0,0 +1,25 @@
+# @fluentui/react-icon-provider
+
+**Icon override provider for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+
+This component is used for providing overrides to the icons in `@fluentui/react-icons-mdl2` and `@fluentui/react-icons`.
+Below is an example of how to do this:
+
+```js
+import { IconProvider, createIconOverride } from '@fluentui/react-icon-provider';
+import { FilterIcon } from '@fluentui/react-icons-mdl2';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
I wonder if using FontAwesome for the demo here might unnecessarily confuse things. Might be better to just make a super basic SVG inside a React component and demo the overrides with that.

Could even do something as generic as this:
```tsx
const FakeIcon = () => <svg>{/* custom SVG content here */}</svg>;

const override = createIconOverride({
  FilterIcon: <FakeIcon />,
});
```